README.md

```diff
@@ -90,19 +90,19 @@ yarn dev
 
 Add in you `.env` in root dir with:
 
-```yaml
+```shell
 # Develop backend settings:
 LIVEKIT_KEY="devkey"
 LIVEKIT_SECRET="secret"
 ```
 
-Add SFU parameter in your local config `./public/config.yml`:
+Add SFU parameter in your local config `./public/config.json`:
 
-```yaml
-"livekit": {
-    "server_url": "ws://localhost:7880",
-    "jwt_service_url": "http:/localhost:8881"
-  },
+```json
+,
+  "livekit": {
+      "livekit_service_url": "http://localhost:7881"
+    }
```

Comment:
```suggestion
,
{
  "livekit": {
    "livekit_service_url": "http://localhost:7881"
  }
}
```

Reply:
This should probably be `http://localhost:7880" but I'm still having an issue with @dbkr's OpenID stuffs in the SFU code here:
https://github.com/vector-im/element-call/blob/livekit/src/livekit/openIDSFU.ts#L39

so I haven't been able to test that yet.

Which livekit-version do I need to get it running?

Reply:
I think it was you with whom I chatted earlier? Are things clearer now?